client/css/style.css

```diff
@@ -837,12 +837,15 @@ kbd {
 	border-color: #79838c;
 }
 
-#windows select.input {
+#windows .input {
```

Comment:
I wouldn't remove the specificity, as this is messing up a few things as far as I can see:

- The main message input is now off by 1px (that can probably easily fixed)
- The custom CSS box now has crazy lines:

Before | After
--- | ---
<img width="476" alt="screen shot 2018-06-11 at 23 42 20" src="https://user-images.githubusercontent.com/113730/41269071-2f76e708-6dd1-11e8-8c36-29b2d5dda4ca.png"> | <img width="481" alt="screen shot 2018-06-11 at 23 42 30" src="https://user-images.githubusercontent.com/113730/41269072-2f8438e0-6dd1-11e8-8574-c4694938463a.png">


Maybe wait after #2315 to remove the specificity (so I don't have to fix input in there too :D)

Reply:
Also even with https://github.com/thelounge/thelounge/wiki/CSS-Modifications#bigger-custom-css-field now, custom CSS box is always 150px because this 35px get higher priority than the height in this snippet (the only one I actually use from the wiki 😂) and therefore the min 150px height applies.